tests/src/e2e/ledger_tests.rs

```diff
@@ -401,3 +402,74 @@ fn invalid_transactions() -> Result<()> {
     client.assert_success();
     Ok(())
 }
+
+/// Test that multiple txs submitted in the same block all get the tx result.
+///
+/// In this test we:
+/// 1. Run the ledger node with 10ms consensus timeout
```

Comment:
```suggestion
/// 1. Run the ledger node with 10s consensus timeout
```

Reply:
I did it again T.T

Reply:
I guess you just love living life in the fast lane :rofl: